Gems/Atom/Tools/MaterialCanvas/Assets/MaterialCanvas/GraphData/Nodes/MaterialOutputs/BasePBR/MaterialGraphName_DepthPass.shader

```diff
@@ -12,10 +12,6 @@
         {
           "name": "MainVS",
           "type": "Vertex"
-        },
-        {
-          "name": "MainPS",
-          "type": "Fragment"
```

Comment:
Why remove this?

Reply:
Because this version of the depth shader does not use the pixel shader. There is a separate depth and shadow shader, which uses the same source file and includes the pixel shader.